pkgs/applications/misc/octoprint/plugins.nix

```
@@ -399,6 +399,27 @@ in {
     };
   };
 
+  octolapse = buildPlugin rec {
+    pname = "Octolapse";
+    version = "0.4.1";
+
+    src = fetchFromGitHub {
+      owner = "FormerLurker";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "13q20g7brabplc198jh67lk65rn140r8217iak9b2jy3in8fggv4";
+    };
+
+    propagatedBuildInputs = with super; [ awesome-slugify setuptools pillow sarge six psutil file_read_backwards ];
+
+    meta = with lib; {
+      description = "Stabilized timelapses for Octoprint";
+      homepage = "https://github.com/FormerLurker/OctoLapse";
+      license = licenses.agpl3;
```

According to the file headers (e.g., [1](https://github.com/FormerLurker/Octolapse/blob/master/octoprint_octolapse/data/lib/c/parsed_command.cpp) and [2](https://github.com/FormerLurker/Octolapse/blob/master/octoprint_octolapse/test/test_settings.py)) the license is `AGPL3+`.

```suggestion
      license = licenses.agpl3Plus;
```